t/OpenTracing/Implementation/DataDog/Client/19_send_span.t

```diff
@@ -78,6 +78,8 @@ subtest "Create a span and capture the request" => sub {
     
     ok $send_ok, "... which returned okay";
     
+    undef $datadog_client;
```

Comment:
Why did this become necessary?

Reply:
Since the created span we use for testing has a `child_of => $some_span_context,`, and thus not being a root span, nor having set a number for the threshold (which defaults to 100) the spans won't be sent.

Doing a `undef $datadog_client` automagically also test the `DEMOLISH`

Reply:
I have added two more tests, to actually include that behavior